src/js/components/Button/StyledButtonKind.js

```diff
@@ -68,36 +79,69 @@ const basicStyle = props => css`
   }
 `;
 
+// could replace this with a call in lodash
```

Comment:
We've tried to reduce the number of dependencies and just keep simple enough things like this inline.

Reply:
Removed the comment and kept the function.